src/lib/sidebar/PerSchemeControls.svelte

```diff
@@ -123,47 +126,87 @@
       return set;
     });
   }
+
+  function onKeyDown(e: KeyboardEvent) {
+    if (showEditModal && e.key == "Escape") {
+      e.stopPropagation();
+      showEditModal = false;
+    }
+  }
 </script>
 
-<h3>
-  {cfg.schemeName($gjSchemes.schemes[scheme_reference])}
+<svelte:window on:keydown={onKeyDown} />
+
+<CheckboxGroup small>
+  <Checkbox bind:checked={showScheme} on:change={showOrHide}>
+    <b>{cfg.schemeName($gjSchemes.schemes[scheme_reference])}</b>
+  </Checkbox>
+</CheckboxGroup>
+
+<div style="display: flex; justify-content: space-between;">
```

Comment:
I'm still a little undecided if we should hide even more when collapsed
![image](https://github.com/user-attachments/assets/9a2ec5ec-e5e7-4b59-8b9e-d6d44cca7d13)